account_financial_report/report/general_ledger.py

```diff
@@ -91,9 +91,12 @@ class GeneralLedgerReportAccount(models.TransientModel):
     initial_debit = fields.Float(digits=(16, 2))
     initial_credit = fields.Float(digits=(16, 2))
     initial_balance = fields.Float(digits=(16, 2))
+    currency_name = fields.Char()
```

Comment:
@hveficent Can you please check my comment, to be consistent we should have here a many2one for currency...and for all modifications, not a lot to be changed...Thanks.